src/lua/compat.lua

```diff
@@ -44,6 +44,13 @@ channel close.
 https://github.com/tarantool/tarantool/wiki/compat%3Afiber_channel_close_mode
 ]]
 
+local SQL_SEQ_SCAN_DEFAULT_BRIEF = [[
+Whether seq_seq_scan session setting should be set to true or false during
+initialization or new session creation.
+
+https://github.com/tarantool/tarantool/wiki/compat%3Asql_seq_scan_default
```

Comment:
We plan to change it to `https://tarantool.io/compat/<option_name>`. See #8325.

Reply:
I think the PR that gets pushed first wins. Another PR should be corrected accordingly :).

Reply:
Okay :) I'll left this comment open, because it is open question, which PR will win.

Reply:
I pushed #8325. Please, adjust the link.

Reply:
Seems resolved. @ImeevMA, thanks!